setup.py

```diff
@@ -23,7 +23,7 @@ def package_data(pkg, root_list):
 
 setup(
     name='xblock-drag-and-drop-v2',
-    version='2.2.10',
+    version='2.2.11',
     description='XBlock - Drag-and-Drop v2',
     packages=['drag_and_drop_v2'],
     install_requires=[
```

Comment:
Does `bleach` need to be added to `install_requires`?

Reply:
> Does `bleach` need to be added to `install_requires`?

It is now imported by the block so makes sense to add here?

Reply:
It is already in requirements.txt but I now also added it into setup.py

Reply:
@davestgermain @xitij2000  It is already in requirements.txt but I now also added it into setup.py